src/control/cmd/daos_agent/infocache_test.go

```diff
@@ -103,7 +106,10 @@ func TestInfoCacheInit(t *testing.T) {
 		{Provider: "ofi+sockets", DeviceName: "eth3_node2", NUMANode: 2}}
 
 	aiCache := attachInfoCache{log: log, enabled: enabled}
-	err := aiCache.initResponseCache(&mgmtpb.GetAttachInfoResp{}, scanResults)
+	netCtx, err := netdetect.Init()
+	defer netdetect.CleanUp(netCtx)
+	common.AssertTrue(t, err == nil, "failed to init netdetect context")
+	err = aiCache.initResponseCache(netCtx, &mgmtpb.GetAttachInfoResp{}, scanResults)
 	common.AssertEqual(t, err, nil, "initResponseCache error")
```

Comment:
Since this initialization step happens before every test, I think it's a good idea to break it out into a helper function that can be called by all the tests. Could have it return the cleanup function so the caller can clean up the ctx afterward.

Reply:
done